lib/SILOptimizer/SILCombiner/SILCombine.cpp

```diff
@@ -690,3 +693,29 @@ void SwiftPassInvocation::eraseInstruction(SILInstruction *inst) {
     }
   }
 }
+
+static llvm::cl::opt<std::string> CondFailConfigFile(
```

Comment:
Can you add a comment here, e.g. the PR description?

Reply:
Will do in a follow-up

Reply:
https://github.com/swiftlang/swift/pull/80127